### ISSUE TITLE
Timed out while loading Accessibility.

### ISSUE THREAD

**Issue by sagar-limbani** [appium/appium#16480]
## The problem

When I'm trying to install .app in simulator receiving “[info] [XCUITest] xcodebuild exited with code ‘65’ and signal ‘null’”  and
Failed to initialize for UI testing: Error Domain=com.apple.dt.XCTest.XCTFuture Code=1000 "Timed out while loading Accessibility." UserInfo={NSLocalizedDescription=Timed out while loading Accessibility.}

Can you please guide me how to resolve this issue.

## Environment

Mac OS : 12.1
Xcode: 13.2.1
Appium: 1.22

## Link to Appium logs

https://gist.github.com/sagar-limbani/e1daf41059f3d303c6bf12efd4112a91

**Comment by mykola-mokhnach** [comment]
Unfortunately this error is coming from XCTest, so we cannot do nothing to workaround it. Consider reporting the issue to Apple.

**Comment by mykola-mokhnach** [comment]
Closed as third party issue
